Mathlib/SetTheory/Game/PGame.lean

```diff
@@ -1338,25 +1338,63 @@ between them. -/
 def toRightMovesNeg {x : PGame} : x.LeftMoves ≃ (-x).RightMoves :=
   Equiv.cast (rightMoves_neg x).symm
 
-theorem neg_moveLeft {x : PGame} (i) : (-x).moveLeft i = -x.moveRight (toLeftMovesNeg.symm i) := by
+@[simp]
+theorem neg_moveLeft {x : PGame} {i} :
+    (-x).moveLeft i = -x.moveRight (toLeftMovesNeg.symm i) := by
   cases x
   rfl
 
-theorem neg_moveRight {x : PGame} (i) :
+@[simp]
+theorem neg_moveRight {x : PGame} {i} :
     (-x).moveRight i = -x.moveLeft (toRightMovesNeg.symm i) := by
   cases x
   rfl
 
-@[simp]
-theorem neg_moveLeft' {x : PGame} (i) :
+theorem neg_moveLeft' {x : PGame} {i} :
     (-x).moveLeft (toLeftMovesNeg i) = -x.moveRight i := by
   simp [neg_moveLeft]
 
-@[simp]
-theorem neg_moveRight' {x : PGame} (i) :
+theorem neg_moveRight' {x : PGame} {i} :
     (-x).moveRight (toRightMovesNeg i) = -x.moveLeft i := by
   simp [neg_moveRight]
 
+@[deprecated neg_moveLeft' (since := "2024-08-27")]
+theorem moveLeft_neg {x : PGame} (i) : (-x).moveLeft (toLeftMovesNeg i) = -x.moveRight i := by
```

Comment:
What exactly is going on here? If you want to switch up the existing lemmas on `moveLeft` and `moveRight`, I recommend you do it in a separate PR.